device/ethernet.yaml

```diff
@@ -82,4 +82,5 @@ components:
           description: Name of the VxLAN instance ( or VxLAN tunnel ) that this Ethernet interface is connected to.
```

Comment:
I think as per the RFC 7348 it is VXLAN and not VxLAN.  Please update in all locations.

Reply:
updated in all locations